arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -4761,12 +4788,15 @@
 			attributes = {
 				KnownAssetTags = (
```

Comment:
It seems there are extra tags included. From the dependency array I only see `ContingentValuesBirdNests` and `fillmoreTopographic`.